### PR TITLE
feat: add content es/bare_metal_machine

### LINKED ISSUE #566: [Spanish] localize 'Bare Metal Machine'
Localize the `content/en/bare_metal_machine.md` document to Spanish

### PR DESCRIPTION
**Pull request by electrocucaracha**
Closes #566

### CODE REVIEW

**Comment by jihoon-seo** [comment]
Preview: https://deploy-preview-689--cncfglossary.netlify.app/es/bare_metal_machine/

Feedback section is displayed like this:
image

@electrocucaracha (or another `es` contributor), Could you add some entries to [`es.toml` of `dev-es` branch](https://github.com/cncf/glossary/blob/dev-es/i18n/es.toml), by opening another PR?

Examples:
- https://github.com/cncf/glossary/blob/540e516abc4950f80463e3997d09355d6a671444/i18n/en.toml#L63-L71
- https://github.com/cncf/glossary/pull/665/files#diff-fd5507f0042e10570f36b7804477e5502fbb6f6a159db8e0a5e86f80c981be06
- https://github.com/cncf/glossary/pull/519/files#diff-bf91abcf6a5d6fc7b18204067500f96305db8352586d24a549a23cd973daa5a5

PTAL! 😊

File: content/es/bare_metal_machine.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Máquina Bare Metal
+status: Completed
+category: technology
+---
+
+## Que es
```
Comment (raelga) [inline content/es/bare_metal_machine.md:7]:
```suggestion
## ¿Qué es?
```

File: content/es/bare_metal_machine.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Bare metal se refiere a una computadora física, más específicamente un servidor, que tiene un solo sistema operativo. La distinción es importante en la informática moderna porque muchos, si no es que la mayoría, de los servidores son [máquinas virtuales](/virtual_machine/). Un servidor físico suele ser una computadora bastante grande con un potente hardware incorporado. La instalación de un sistema operativo y la ejecución de aplicaciones directamente en ese hardware físico, sin [virtualización](/virtualization/), se conoce como ejecución "bare metal".
+
+## Problema que aborda
+
+Emparejar un sistema operativo con una computadora física es el patrón original de la informática. Todos los recursos de la computadora física están disponibles directamente para el sistema operativo y sin la presencia de una capa de virtualización, no hay un retraso artificial en la traducción de las instrucciones del sistema operativo al hardware.
+
+## Cómo ayuda
```
Comment (raelga) [inline content/es/bare_metal_machine.md:15]:
```suggestion
## ¿Cómo ayuda?
```

**Review by raelga — APPROVED** [review]

File: content/es/bare_metal_machine.md (hunk 1 above)
Comment (electrocucaracha) [reply content/es/bare_metal_machine.md]: Fixed on 2c9bc46

File: content/es/bare_metal_machine.md (hunk 2 above)
Comment (electrocucaracha) [reply content/es/bare_metal_machine.md]: Fixed on 2c9bc46

**Review by raelga — APPROVED** [review]

File: content/es/bare_metal_machine.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## ¿Cómo ayuda?
+
+Al dedicar todos los recursos informáticos de una computadora a un solo sistema operativo, potencialmente se proporciona el mejor rendimiento posible al sistema operativo. Si necesita ejecutar una carga de trabajo que debe tener un acceso extremadamente rápido a los recursos de hardware, bare metal puede ser la solución adecuada.
+
+En el contexto de las [aplicaciones nativas de la nube](/cloud_native_apps/), generalmente pensamos en el rendimiento en términos de [escalado](/scalability/) para una gran cantidad de eventos simultáneos, que pueden manejarse mediante el [escalado horizontal](/horizontal_scaling/) (agregando más máquinas a su grupo de recursos). Sin embargo, algunas cargas de trabajo pueden requerir [escalado vertical](/vertical_scaling/) (agregar más potencia a una máquina física existente) y/o una respuesta extremadamente rápida de hardware físico, en cuyo caso se adapta mejor el bare metal. Bare metal también le permite ajustar el hardware físico y posiblemente incluso los controladores de hardware para ayudarlo a realizar su tarea.
```
Comment (krol3) [inline content/es/bare_metal_machine.md:19]:
```suggestion
En el contexto de las [aplicaciones nativas de la nube](/cloud_native_apps/), generalmente pensamos en el rendimiento en términos de [escalabilidad](/scalability/) para una gran cantidad de eventos simultáneos, que pueden manejarse mediante la [escalabilidad horizontal](/horizontal_scaling/) (agregando más máquinas a su grupo de recursos). Sin embargo, algunas cargas de trabajo pueden requerir [escalado vertical](/vertical_scaling/) (agregar más potencia a una máquina física existente) y/o una respuesta extremadamente rápida de hardware físico, en cuyo caso se adapta mejor el bare metal. Bare metal también le permite ajustar el hardware físico y posiblemente incluso los controladores de hardware para ayudarlo a realizar su tarea. 
```

**Review by krol3 — COMMENTED** [review]
Algunas sugerencias, escuche poco el termino "escalado", es más común las palabras escalabilidad.

**Review by krol3 — APPROVED** [review]
LGTM

File: content/es/bare_metal_machine.md (hunk 3 above)
Comment (electrocucaracha) [reply content/es/bare_metal_machine.md:19]: @krol3 que te parece esta opcion?

```suggestion
En el contexto de las [aplicaciones nativas de la nube](/cloud_native_apps/), generalmente pensamos en el rendimiento en términos de [escalabilidad](/scalability/) para una gran cantidad de eventos simultáneos, que pueden mitigarse mediante el [escalado horizontal](/horizontal_scaling/) (agregando más máquinas a su grupo de recursos). Sin embargo, algunas cargas de trabajo pueden requerir [escalado vertical](/vertical_scaling/) (agregar más potencia a una máquina física existente) y/o una respuesta extremadamente rápida de hardware físico, en cuyo caso se adapta mejor el bare metal. Bare metal también le permite ajustar el hardware físico y posiblemente incluso los controladores de hardware para ayudarlo a realizar su tarea. 
```

**Review by krol3 — APPROVED** [review]
Thanks @electrocucaracha

**Review by raelga — APPROVED** [review]

**Comment by electrocucaracha** [comment]
@seokho-son can you help us to merge this PR?

**Comment by seokho-son** [comment]
@electrocucaracha I noticed this request now. Thank you for updating i18n/es.toml !!

**Review by seokho-son — APPROVED** [review]
lgtm!

**Pull request merged**